slot/src/slot/slot.providers.ts

```diff
@@ -0,0 +1,8 @@
+import { DataSource } from "typeorm";
+import { Slot } from "./slot.entity";
+
+export const slotProviders = [{
```

Comment:
not a good idea , why not to have separate repo file or even some storage service  which will encapsulate the functionality of repo with db